Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos/ruby-24-centos7
+FROM centos/ruby-25-centos7
```

Comment:
I would use UBI instead of RHSCL CentOS

Reply:
Good point. 

`registry.access.redhat.com/rhscl/ruby-25-rhel7:2.5` is UBI-based.

Reply:
@gsaslis we tried but without success 😞 

Our system-builder needs a lot of xdg- packages to have chrome installed on it to be used on CI stuffs. UBI images don't have it and neither the package available to install. We can install it using centos packages but we can't guarantee to not make a mess in the packaging stuff of RHEL.

Reply:
@duduribeiro do you have that list of packages missing from UBI images handy anywhere?

Reply:
@gsaslis AFAIK, these:

```
	libXss.so.1()(64bit) is needed by google-chrome-stable-63.0.3239.108-1.x86_64
	libatk-1.0.so.0()(64bit) is needed by google-chrome-stable-63.0.3239.108-1.x86_64
	libgconf-2.so.4()(64bit) is needed by google-chrome-stable-63.0.3239.108-1.x86_64
	libgdk-3.so.0()(64bit) is needed by google-chrome-stable-63.0.3239.108-1.x86_64
	libgdk_pixbuf-2.0.so.0()(64bit) is needed by google-chrome-stable-63.0.3239.108-1.x86_64
	libgtk-3.so.0()(64bit) is needed by google-chrome-stable-63.0.3239.108-1.x86_64
	xdg-utils is needed by google-chrome-stable-63.0.3239.108-1.x86_64
```